streamer/src/quic.rs

```diff
@@ -537,7 +555,8 @@ pub fn spawn_server(
                 }
 
                 if let Ok(Some(connection)) = timeout_connection {
-                    if let Ok(new_connection) = connection.await {
+                    let result = connection.await;
+                    if let Ok(new_connection) = result {
```

Comment:
necessary?

Reply:
It was used to dump out the error reason at line 631